TypeCobol.LanguageServer/TypeCobolCustomLanguageServer/TypeCobolCustomLanguageServer.cs

```diff
@@ -52,7 +52,8 @@ protected override InitializeResult OnInitialize(InitializeParams parameters)
         /// <param name="options">Client's Options</param>
         protected override void OnDidChangeConfiguration(string[] options)
         {
-            this.UseOutlineRefresh = options.Contains("-ol");
+            if (options.Contains("-team"))
+                this.UseOutlineRefresh = options.Contains("-ol");
```

Comment:
As our client doesn't pass option `-ol`, just invert the logic. It would make more sense to rename the param to "Deactivate Outline Refresh" : `-dol`.
```suggestion
                this.UseOutlineRefresh = !options.Contains("-dol");
```

Reply:
Sorry doing like that makes most LSR test fail, this is not the moment to review the majority of LSR tests, the -team option is here the alternative option. 
Because doing like that will by default allow outilne on all LSR tests

Reply:
Just change the logic in LSRTestHelper.cs:
```C#
           testOptions += useOutline ? ",\"-ol\"" : "";
```
into
```C#
           testOptions += useOutline ? "":  ",\"-dol\"";
```
Then every tests will pass.

Or you can imagine a more complete option:
without parameter `-ol` : outline is activated by default
`-ol=true` : activate outline
`-ol=false` : activate outline

Anyway goals are:
- Don't introduce a specific option for Euro-Information if we don't need to
- The rule about our team is temporary and is not a security rule:
   -  We just want to hide complex options for the user. 
   - We might decide to open this option for every users in the future.
- Our current TypeCobol client allows users to de/activate Outline and the `-team` will break this behavior.
- The rule is already enforced on the client side, no need to duplicate it on the server side (if it was a security or business rule that would be different)
- Fall back to the behavior of 1.4.x : where Outline is activated by default

Reply:
Finally the combination of -dol an -ol options is the good choice for non-regression with v1.4.
done in 3bb41af.